deps.bzl

```diff
@@ -1487,9 +1487,9 @@ def install_buildbuddy_dependencies(workspace_name = "buildbuddy"):
             "gazelle:resolve go github.com/containernetworking/cni/pkg/types/current @com_github_containernetworking_cni//pkg/types/current",
         ],
         importpath = "github.com/firecracker-microvm/firecracker-go-sdk",
-        replace = "github.com/bduffany/firecracker-go-sdk",
-        sum = "h1:gxdryq8e81djt9+N7gUa4FVJO427la9UUQEiMo2B8tE=",
-        version = "v0.0.0-20230702173145-62f8e172aec2",
+        replace = "github.com/maggie-lou/firecracker-go-sdk",
```

Comment:
can we fork this under the buildbuddy org instead of ping ponging between personal accounts?

Reply:
it's a good idea